Form-Controls/styles.css

```diff
@@ -0,0 +1,66 @@
+* {
+    padding: 5px;
+    margin: 10px;
+    background-color: rgb(222, 206, 206);
```

Comment:
assigning a background color to all elements might get tricky -- what's a way we can style the layout's background without doing so for every element?